src/pages/LandingPage/components/EventList/styled.ts

```diff
@@ -18,7 +18,7 @@ export const GridItem = styled.li`
   }
 
   @media screen and (min-width: 1250px) {
-    width: 29vw;
+    width: 28vw;
```

Comment:
This isn't very scientific, but at `29` layout wasn't correct, but `28` looked OK for the demo

Reply:
Typical 😅